json/src/test/kotlin/ktx/json/serializerTest.kt

```diff
@@ -0,0 +1,78 @@
+package ktx.json
+
+import com.badlogic.gdx.utils.Json
+import com.badlogic.gdx.utils.JsonValue
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import io.kotlintest.matchers.shouldEqual
+import io.kotlintest.matchers.shouldThrow
+import org.junit.Test
+
+@Suppress("unused")
+class `should implement ReadOnlySerializer with no 'write' method overriden`<T> : ReadOnlySerializer<T> {
+  // Guarantees that [write] method is optional to implement.
+
+  override fun read(json: Json, jsonData: JsonValue, type: Class<*>?): T {
+    throw NotImplementedError()
+  }
```

Comment:
You can also write a test that verifies the exception thrown by `write` method of this class.

Reply:
Not sure the way I did it is OK, but I can't see how to do it the other way.